_sips/process-specification.md

```diff
@@ -288,6 +288,8 @@ have the following purposes:
 - Make any exception to the process that they judge necessary to unblock a 
   situation.
 
+If a Committee Member cannot attend a meeting, they are welcome to share their feedback about the proposals listed in the agenda of the meeting with the Chairperson, who will relate it during the meeting. A Committee member cannot give their voting power to someone else, and they must not miss more than 2 meetings within a year.
```

Comment:
the “must not” wording here is very strong and unequivocal, but the consequences aren't clear